src/styles/blocks/catalog.scss

```
@@ -12,11 +12,17 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-property: color, transform $transition-time;
```

```suggestion
    transition: color $transition-time, transform $transition-time;
```
Or you can add transition-duration property separately

---

работает если записать отдельным свойством